app/validators/sections/section_validator.py

```diff
@@ -40,11 +40,28 @@ def validate_repeat(self):
             self.validate_list_exists(section_repeat["for_list"])
 
     def validate_summary(self):
-        section_summary = self.section.get("summary", None)
+        if not (section_summary := self.section.get("summary", None)):
+            return
+        for item in section_summary.get("items", []):
+            self.validate_list_exists(item.get("for_list"))
 
-        if section_summary:
-            for item in section_summary.get("items", []):
-                self.validate_list_exists(item.get("for_list"))
+        for_lists = []
+        content_block_in_section = False
+
+        for block_id in self.questionnaire_schema.get_section_block_ids(
+            self.section.get("id")
+        ):
+            block = self.questionnaire_schema.get_block(block_id)
+            if (
+                block.get("type") in ["ListCollector", "ListCollectorContent"]
+                and block.get("for_list") not in for_lists
```

Comment:
this should exist if its a collector type so would change to `block["for_list"]`